docs/reference/snapshot-restore/apis/create-snapshot-api.asciidoc

```diff
@@ -0,0 +1,190 @@
+[[create-snapshot-api]]
+=== Create snapshot API
+++++
+<titleabbrev>Create snapshot</titleabbrev>
+++++
+
+Takes a <<snapshot-restore,snapshot>> of a cluster or specified data streams and
+indices.
+
+////
+[source,console]
+-----------------------------------
+PUT /_snapshot/my_repository
+{
+  "type": "fs",
+  "settings": {
+    "location": "my_backup_location"
+  }
+}
+-----------------------------------
+// TESTSETUP
+////
+
+[source,console]
+-----------------------------------
+PUT /_snapshot/my_repository/my_snapshot
+-----------------------------------
```

Comment:
It looks like the docs CI is failing because this snapshot doesn't finish before the example at the bottom. You can spoof the `wait_for_completion` query param using a `// TEST` comment.
```suggestion
[source,console]
-----------------------------------
PUT /_snapshot/my_repository/my_snapshot
-----------------------------------
// TEST[s/my_snapshot/my_snapshot?wait_for_completion=true/]
```